plugins/Jobs.cpp

```diff
@@ -566,14 +570,16 @@ void BedrockJobsCommand::process(SQLite& db) {
 
             // Are we creating a new job, or updating an existing job?
             if (updateJobID) {
-                // Update the existing job.
-                if(!db.writeIdempotent("UPDATE jobs SET "
-                                         "repeat   = " + SQ(SToUpper(job["repeat"])) + ", " +
-                                         "data     = JSON_PATCH(data, " + safeData + "), " +
-                                         "priority = " + SQ(priority) + " " +
-                                       "WHERE jobID = " + SQ(updateJobID) + ";"))
-                {
-                    STHROW("502 update query failed");
+                if (!SContains(job, "overwrite") || job["overwrite"] == "true") {
```

Comment:
I think you also want to check if `overwrite` is `==` and treat it as true too?

Reply:
I'm not quite following. What would we check if `overwrite` is `==` to?

Reply:
Damn sorry, I mean is equal to `''`

Reply:
Ah good catch, will add that